app/infra/notificationsService/notificationsService.js

```diff
@@ -18,6 +20,20 @@ class NotificationsService {
       };
     }
   };
+
+  static checkAppVisibility() {
+    ipcRenderer.send(ipcConsts.CHECK_APP_VISIBLITY);
+    return new Promise<string, Error>((resolve: Function) => {
```

Comment:
Response is binary: true or false. Why there are two separate channels for processing the response?

Reply:
Will consolidate.